content/02.main-text.md

```diff
@@ -172,7 +172,7 @@ Because all bitcoin transactions compete for limited space on the blockchain, th
 OpenTimestamps avoids this fee by encoding many timestamps into a single Bitcoin transaction [@url:https://petertodd.org/2016/opentimestamps-announcement].
 There can be a lag of a few hours before the transaction is made, which is suitable for the purposes of scientific writing.
 
-Manubot has the potential to transform publication, making it permissionless, reproducible, free of charge, and largely open source.
+Manubot and its dependencies are free of charge and largely open source.
```

Comment:
This was redundant with the new closing paragraph and I found the original phrasing confusing because Manubot itself is fully open source.